unittests/test_utils.py

```diff
@@ -30,3 +30,11 @@ def test_raise_for_status_server_error():
 
 def test_raise_for_status_no_error():
     utils.raise_for_status(MockResponse(status_code=HTTPStatus.OK))
+
+
+def test_fix_path_sep_for_current_platform_windows_path():
+    assert utils.fix_path_sep_for_current_platform(r"a\b\c") == "a/b/c"
```

Comment:
I'd suggest using `os.path.join(...)` instead of `a/b/c`.
Then the test will be perfectly portable, and in case we want ever to run in on Windows as well - zero-cost.

Reply:
right, but then I can't test windows paths on linux

Reply:
Fixed